doc/UsersGuide/BackgroundInfo/Introduction.rst

```diff
@@ -117,8 +141,6 @@ A list of available component documentation is shown in :numref:`Table %s <list_
      - Location
    * - spack-stack Documentation
      - https://spack-stack.readthedocs.io/en/latest/
-   * - HPC-Stack Documentation
-     - https://hpc-stack.readthedocs.io/en/latest/
    * - UFS_UTILS Technical Documentation
      - https://noaa-emcufs-utils.readthedocs.io/en/latest
    * - UFS_UTILS Scientific Documentation
```

Comment:
@MichaelLueken Do you think these links should get updated to the ones used in the release because even `develop` isn't using the latest UFS_UTILS? Like, are the SRW v3.0.0 links more accurate for develop currently?

Reply:
@jdkublnick I think the CCPP Scientific doc link should be the v3.0.0 link (instead of the v2.2.0 link) because there isn't a develop branch version, so we use the most recent. GitHub won't let me comment that line separately tho since it doesn't have any edits.

Reply:
@gspetro-NOAA @jdkublnick Yes, I would recommend using the SRW v3.0.0 links for UFS_UTILS.  The SRW App is currently using a version of UFS_UTILS from February 6, 2024, so we should not be pointing to the latest documentation in NOAA-EMC/UFS_UTILS.  The NOAA-EPIC/UFS_UTILS documentation from the SRW v3.0.0 release is up-to-date with the SRW App.